run_all_tests.sh

```diff
@@ -7,18 +7,9 @@
 # Fail immediately in case of errors and/or unset variables
 set -eu -o pipefail
 
-# todo: All Java tests (Unit & ITs) are skipped until lib loading is fixed [https://jira.bf.local/browse/ECR-942].
-# Run unit and integration tests in ci-build profile. This profile includes:
-#  - Java unit & integration tests, including ci-only & slow non-critical tests,
-#    which are excluded in the default profile.
-#  - Checkstyle checks as errors.
-#  - Native unit & integration tests that do not require a JVM.
-# See build definitions of the modules for more.
-mvn install \
-  -DskipTests \
-  --activate-profiles ci-build \
-  -Drust.compiler.version="stable"
+# Run all java tests and native unit tests.
+./run_maven_tests.sh
 
-# Run native integration tests that require a JVM.
+# Run native integration tests that require prepared classpaths for fake classes.
```

Comment:
require ejb-fakes module and/or JVM.

Reply:
JVM we need not only for ITs now.

Reply:
Yes, but ITs require either one or both.

Reply:
All tests require JVM, it makes no sense to mention it separately here.